identity-account/src/account/publish_options.rs

```diff
@@ -25,6 +25,7 @@ impl PublishOptions {
   /// for identities with many updates.
   ///
   /// See the IOTA DID method specification for more details.
+  #[deprecated(since = "0.5.0", note = "diff chain features are slated for removal")]
   #[must_use]
   pub fn force_integration_update(mut self, force: bool) -> Self {
     self.force_integration_update = force;
```

Comment:
Shouldn't this just be removed? If the account always publishes integration updates, then this flag becomes useless. Compared to `0.4` this wouldn't even be breaking, because we added `PublishOptions` after that release.
Raises the question whether our `Breaking Change` label is relative to the previous release or relative to the last "proper" release (`0.4` in this case). Would be weird actually, to add a method that is deprecated from the start.

As far as I can see only the `test_account_publish_options_force_integration` would have to be removed, so it shouldn't have to many compounding effects.

Reply:
The `force_integration_update` still has the effect of force-publishing even if there are no changes, so I'm unsure if we should remove it entirely or replace it with something like `force_publish`. The point is that because it was decided not to remove the diff chain (yet), there is the chance that all these features will be un-deprecated, which would make `force_integration_update` immediately useful again as it currently stands.

`Breaking change` is relative to the last release.